diff -Naur gst-python-0.8.1.orig/configure.ac gst-python-0.8.1/configure.ac
--- gst-python-0.8.1.orig/configure.ac	2004-12-20 22:20:01.667531728 +0100
+++ gst-python-0.8.1/configure.ac	2004-12-20 22:31:38.169647312 +0100
@@ -67,12 +67,12 @@
 AC_MSG_RESULT($PYGTK_DEFSDIR)
 
 AC_MSG_CHECKING(for pygtk h2def)
-PYGTK_H2DEF=`$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.py
+PYGTK_H2DEF=`$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.pyc
 AC_SUBST(PYGTK_H2DEF)
 AC_MSG_RESULT($PYGTK_H2DEF)
 
 AC_MSG_CHECKING(for pygtk codegen)
-PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.py"
+PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.pyc"
 dnl PYGTK_CODEGEN="$PYTHON \$(top_srcdir)/codegen/codegen.py"
 AC_SUBST(PYGTK_CODEGEN)
 AC_MSG_RESULT($PYGTK_CODEGEN)
