--- gst-python-1.16.0/acinclude.m4~	2018-12-31 00:47:50.000000000 +0100
+++ gst-python-1.16.0/acinclude.m4	2019-11-04 00:28:59.008572417 +0100
@@ -79,7 +79,7 @@
 
 dnl deduce PYTHON_LIBS
 if $PYTHON-config --help 1>/dev/null 2>/dev/null; then
-  PYTHON_LIBS=`$PYTHON-config --ldflags 2>/dev/null`
+  PYTHON_LIBS=`$PYTHON-config --ldflags --embed 2>/dev/null ||  $PYTHON-config --ldflags 2>/dev/null`
   PYTHON_LIB=`$PYTHON -c "import distutils.sysconfig as s; print(s.get_python_lib(standard_lib=1))"`
   if echo "$host_os" | grep darwin >/dev/null 2>&1; then
     dnl OSX is a pain. Python as shipped by apple installs libpython in /usr/lib
